### PR TITLE
Add timezone_minute Presto function

### PR DESCRIPTION
**Pull request by Ann-Rose**
Resolves https://github.com/facebookincubator/velox/issues/4521

### CODE REVIEW

**Comment by ashokku2022** [comment]
@aditi-pandit Please review when you get time, thanks!

File: velox/functions/prestosql/DateTimeFunctions.h [hunk 1]
```diff
@@ -1041,4 +1041,31 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
```
Comment (Yuhta) [inline velox/functions/prestosql/DateTimeFunctions.h:1048]: This probably should share the same implementation with #4520

Comment (Ann-Rose) [reply velox/functions/prestosql/DateTimeFunctions.h:1048]: @Yuhta Can we push this separately for this once. For future functions will implement the similar ones in the same PR.

Comment (majetideepak) [reply velox/functions/prestosql/DateTimeFunctions.h:1087]: @Ann-Rose once #4520 lands, let's refactor and re-use the code to implement this function.

**Comment by Ann-Rose** [comment]
@Yuhta @majetideepak https://github.com/facebookincubator/velox/pull/4520 was merged. I have refactored the code to reuse it for timezone_minute. Can you please review

**Pull request closed without merging**